packages/api-utils/lib/getSlug.js

```diff
@@ -4,14 +4,17 @@ const require = createRequire(import.meta.url);
 
 const { slugify } = require("transliteration");
 
+const standardSlug = "a-zA-Z0-9-";
```

Comment:
The default `allowedChars` based on the transliteration documentation is `a-zA-Z0-9-_.~'`